x-pack/plugins/endpoint/server/routes/metadata/query_builders.test.ts

```diff
@@ -34,7 +39,7 @@ describe('query builder', () => {
           aggs: {
             total: {
               cardinality: {
-                field: 'host.id.keyword',
+                field: 'host.id',
```

Comment:
im a bit of a noob about this. can you explain this change? is this just keeping us in sync w/ latest mappings?

Reply:
Yeah it's because the mapping that was being used for metadata was based off a dynamically generated mapping created by default by elasticsearch when no mapping exists. Now that we have solidified and generated a mapping using ECS we want to use that. With the new mapping the field `keyword` no longer exists.

https://www.elastic.co/blog/strings-are-dead-long-live-strings